app/Controller/PagesController.php

```diff
@@ -36,6 +36,46 @@ class PagesController extends AppController {
  * @var array
  */
 	public $uses = array();
+	public $uses2 = array();
+	public $uses3 = array();
+
+/**
+ * TooManyMethods
+*/
```

Comment:
__[code_sniffer]__
* Missing @return tag in function comment
* Expected 1 space(s) before asterisk; 0 found